src/linear_algebra/nonsingular_inverse.lean

```diff
@@ -52,7 +52,6 @@ open_locale matrix
 open equiv equiv.perm finset
 
 -- Increase max depth to allow inference of `mul_action α (matrix n n α)`.
```

Comment:
```suggestion
```

Reply:
Thanks, this one was not catched by my regexp